streams-building/apps/energy-meter-front/src/app/AppLayout.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { useHistory } from 'react-router-dom';
+import {
+  Page,
+  PageHeader,
+  SkipToContent
+} from '@patternfly/react-core';
+import logo from '@app/images/training_white.png';
+
+interface IAppLayout {
+  children: React.ReactNode;
+}
+
+const AppLayout: React.FunctionComponent<IAppLayout> = ({ children }) => {
+  const [isNavOpen, setIsNavOpen] = React.useState(true);
```

Comment:
We're not using nav, we can probably delete these?

Reply:
Yeah, I cleaned some code from the Patternfly seed project, but I missed these. Nice catch!